sbom.cdx.json

```diff
@@ -0,0 +1,43 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "authors": [
+      {
+        "name": "@VCS_SBOM_AUTHORS@"
+      }
+    ]
+  },
+  "components": [
+    {
+      "type": "library",
+      "bom-ref": "pkg:github/Mbed-TLS/mbedtls@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:mbed:mbedtls:@VCS_TAG@:*:*:*:*:*:*:*",
+      "name": "mbedtls",
+      "version": "@VCS_VERSION@",
+      "description": "Implements cryptographic primitives, X.509 certificate manipulation and SSL/TLS and DTLS protocols",
+      "supplier": {
+        "name": "The Mbed TLS Contributors"
```

Comment:
“The Mbed TLS Contributors” is a collective name for all the co-authors of the code. I don't think it can be considered a name for the supplier. The supplier is either the [Mbed TLS organization on GitHub](https://github.com/Mbed-TLS), if you specifically want to refer to the distribution of source code via GitHub, or [Trusted Firmware](https://www.trustedfirmware.org/projects/mbed-tls/), which is the manufacturer in the sense that it's the organization who makes the rules for what goes into the project.

Reply:
I think either `Trusted Firmware` or `Mbed TLS organization` would be fine, but I'm erring more on the former if we do change the CPE vendor. Agree?

Reply:
Done, thanks.